src/App.css

```diff
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  background-color: rgba(0, 0, 0, 0.05);
+  font-family: Montserrat, sans-serif;
+}
```

Comment:
All differences present in this image please try to fix all of them.

ORIGINAL
<img width="700" alt="Screen Shot 2022-04-09 at 11 46 34 PM" src="https://user-images.githubusercontent.com/69011963/162588293-9fb87edb-8297-4c3a-91fe-572ebabf9c20.png">


YOUR DESIGN
<img width="700" alt="Screen Shot 2022-04-09 at 11 31 18 PM" src="https://user-images.githubusercontent.com/69011963/162588301-40bd9d64-2419-48b3-9925-2b82933b8351.png">